FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -9,93 +10,84 @@
-                for(int i = 0; i < columnNames.size(); i++) {
-                    if (!resultSet.isNull(columnNames.get(i))){
-                        Type type = resultSet.getColumnType(columnNames.get(i));
-                        switch (type.getCode()) {
-                            case ARRAY :
-                                //todo
-                                // maybe need specific array type from table (or maybe we can get from type field)
-                                String arrayType = columnTypes.get(i);
-                                if (arrayType.equals("ARRAY<STRING(MAX)>")) {
-                                    row.add(String.valueOf(resultSet.getStringList(columnNames.get(i))));
-                                }
-                                break;
-                            case BOOL:
-                                row.add(String.valueOf(resultSet.getBoolean(columnNames.get(i))));
-                                break;
-                            case BYTES:
-                                row.add(String.valueOf(resultSet.getBytes(columnNames.get(i))));
-                                break;
-                            case DATE:
-                                row.add(String.valueOf(resultSet.getDate(columnNames.get(i))));
-                                break;
-                            case FLOAT64:
-                                row.add(String.valueOf(resultSet.getDouble(columnNames.get(i))));
-                                break;
-                            case INT64:
-                                row.add(String.valueOf(resultSet.getLong(columnNames.get(i))));
-                                break;
-                            case NUMERIC:
-                                row.add(String.valueOf(resultSet.getBigDecimal(columnNames.get(i))));
-                                break;
-                            case STRING:
-                                row.add(resultSet.getString(columnNames.get(i)));
-                                break;
-                            case STRUCT:
-                                //not used at the moment
-                                break;
-                            case TIMESTAMP:
-                                row.add(String.valueOf(resultSet.getTimestamp(columnNames.get(i))));
-                                break;
-                        }
-                    }
-                    else{
+                for (int i = 0; i < columnNames.size(); i++) {
+                    if (resultSet.isNull(columnNames.get(i))) {
                         row.add("");
+                        continue;
+                    }
+                    Type type = resultSet.getColumnType(columnNames.get(i));
+                    switch (type.getCode()) {
+                        case ARRAY:
+                            String arrayType = columnTypes.get(i);
+                            if (arrayType.equals("ARRAY<STRING(MAX)>")) {
```

Comment:
a good habit to have is when we compare a String variable to a String constant, do `constant.equals(var)` instead of `var.equals(constant)`. why? because String objects are nullable, and if for some reason your var is null, you'll get a null pointer exception when calling `equals`.